aio/build.sh

```diff
@@ -142,7 +146,11 @@ cp /mingw64/share/icons/hicolor/scalable/places/*.svg /mingw64/share/icons/gnome
 # build gramps
 rm -rf dist aio/dist
 python setup.py bdist_wheel
-appbuild="r$(git rev-list --count HEAD)-$(git rev-parse --short HEAD)"
+if `grep -q '^DEV_VERSION\s*=\s*True' gramps/version.py`; then
+    appbuild="r$(git rev-list --count HEAD)-$(git rev-parse --short HEAD)"
```

Comment:
`git describe` is perhaps clearer and definitely simpler. Today's gramps60 HEAD produces `v6.0.0-beta1-27-gf5e551c3f`, composed of the tag, the number of commits following the tag, and the shortened sha1 of the commit. If the commit is tagged `git describe` returns just the tag, e.g.:
```
> git describe v6.0.0-beta1
v6.0.0-beta1
```

Reply:
@Nick-Hall do you always tag *before* you do the release build? If so `git describe` looks like a good improvement

Reply:
Yes.  We should probably use `git describe` instead of our existing `get_git_revision` script.  The Gramps version reported from the command line and shown in the About dialog is wrong.

```
> python Gramps.py -v
6.0.0-beta2-149fb610ca
```

I bump the version number after each release and this is misleading.

I'll investiage.

Reply:
Plus `build.sh` just uses `VERSION_TUPLE` so it loses the `beta1` part making it even more misleading.

`master`'s VERSION_TUPLE is 6.0.0 and its last tag is v5.2.4; neither is really right. In that case maybe `"$(git branch)-$(git rev-list --count HEAD)-$(git rev-parse --short HEAD)"` would be most correct... but perhaps we don't build AIOs from `master` so it doesn't matter?

Reply:
The "beta1" part is injected in via a workflow input and used if DEV_VERSION in version.py is False

Reply:
Using `git describe` works well to provide a better description of development versions .  See PR #1938.

Reply:
> The "beta1" part is injected in via a workflow input and used if DEV_VERSION in version.py is False

That misses the point. `6.0.0-r1-5a76f63` as a dev build tag is misleading because there is no 6.0.0 yet.